packages/mui-base/src/SliderUnstyled/SliderValueLabelUnstyled.tsx

```diff
@@ -20,9 +19,13 @@ const useValueLabelClasses = (props) => {
 /**
  * @ignore - internal component.
  */
-export default function SliderValueLabelUnstyled(props) {
-  const { children, className, value } = props;
-  const classes = useValueLabelClasses(props);
+export default function SliderValueLabelUnstyled(props: SliderValueLabelUnstyledProps) {
+  const { children, className, value, open } = props;
+  const classes = useValueLabelClasses(open);
```

Comment:
For consistency, I would propose leaving the implementation as it was before. in all `use*Classes` hooks we send the `ownerState` (all props with default values & states).

Reply:
FYI: This component is removed in #35479 from SliderUnstyled.

Reply:
It's fine, even if the component is moved to the other packages, it's important to have consistency between the components.

Reply:
cc @sai6855 I believe this is the only thing that's missing for merging the pull request.